src/app/ui/components/containers/container.layout.tsx

```diff
@@ -1,3 +1,5 @@
+import { radixBaseCSS } from '@radix-ui/themes/styles.css';
```

Comment:
Moving this here means the UI library will rely on `@radix-ui/themes/` but not the APP. 

Before we were loading this CSS in a weird way in `app.tsx`. 

I'm not sure if this is the best way of doing things but it means we can abstract the `ui` folder out of the application if we want. Then our library uses radix as a base and not the app itself. 

We would have some more work to do moving the global `radixStyles` but it could be worthwhile

Reply:
@kyranjamie @fbwoolf just in case I forget about this / it gets missed. 

I changed how we load this `radixBaseCSS` theme but now as we don't really use it at all, we are actually close to deleting it completely which we should do soon after this PR